test/unit/alphabet/composite/alphabet_variant_test.cpp

```diff
@@ -13,6 +13,7 @@
 #include <seqan3/alphabet/composite/alphabet_variant.hpp>
 #include <seqan3/alphabet/gap/gap.hpp>
 #include <seqan3/alphabet/nucleotide/all.hpp>
+#include <seqan3/core/char_operations/predicate.hpp>
```

Comment:
Why do we need this?

Reply:
Because `seqan3::is_in_alphabet` is defined in this header

Reply:
I disagree :( Because this issue was not because `seqan3::is_in_alphabet` in predicate was wrong but because the alphabet_variant implementation had a flaw. So this is an issue of the alphabet_variant and should therefore be in its test.
Since `seqan3::is_in_alphabet` is only using `seqan3::char_is_valid_for` the test for the latter would suffice and we could remove the dependency on `predicate.hpp`. But we should nor make the predicate test depends on `alphabet_variant.hpp`, `dna4.hpp` and `gapped.hpp`.

Reply:
* What is that regression about? Oh, yeah about  `seqan3::is_in_alphabet` which belongs to the "test/unit/core/char_operations/char_predicate_test.cpp"
* Which test already includes *all* alphabets?
   hint:
   https://github.com/seqan/seqan3/blob/d06f9acae65e01375109f43f0bfa5202520a534f/test/unit/core/char_operations/char_predicate_test.cpp#L11-L14

The regression should be added where the regression belongs to.
Imagine someone who uses `seqan3::is_in_alphabet`, where will he look for tests? Sure, he will look in the `gapped` test, because it is obvious.

Please change this. Thank you!